common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -747,6 +737,21 @@
                     self.isMouseFocus = false;
                     self.rendered = true;
                     self.state.el.addClass('is-captions-rendered');
+
+                    self.subtitlesEl.attr('aria-label', gettext('Video transcript: activating an item in this group will spool the video to the corresponding time point.')) // jshint ignore:line
```

Comment:
This text is for screenreader only users and it describes how the transcript links work. Activating a link in the transcript will skip to that point in time in the video. I believe "spool" was here originally, but I did some shuffling with the code so it appears new.